CMakeLists.txt

```diff
@@ -1706,7 +1710,7 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/libnetcdf.settings.in"
 file(READ "${CMAKE_CURRENT_BINARY_DIR}/libnetcdf.settings"
   LIBNETCDF_SETTINGS)
 message(STATUS ${LIBNETCDF_SETTINGS})
-
+message(STATUS ${DEPR_OPT})
```

Comment:
`message(WARNING)`, we really want to raise the user's attention. I would move it inside the for loop call. I don't remember if a summary is redisplayed when using `WARNING`, but you can use `cmake_language(DEFER)` to move it at the end.